### PR TITLE
JAVA-28508: Fix formatting of POMs Modules D to Q

### PR DESCRIPTION
**Pull request by Bipinkumar27**

### CODE REVIEW

File: microservices-modules/rest-express/pom.xml [hunk 1]
```diff
@@ -2,13 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (anastasiosioannidis) [inline microservices-modules/rest-express/pom.xml:2]: Appropriate order isn't correct:
modelVersion
artifactId
version
Name

File: persistence-modules/spring-data-jpa-repo-3/pom.xml [hunk 2]
```diff
@@ -3,16 +3,17 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (anastasiosioannidis) [inline persistence-modules/spring-data-jpa-repo-3/pom.xml:3]: Add a space between dependencies and properties

File: persistence-modules/spring-jdbc-2/pom.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (anastasiosioannidis) [inline persistence-modules/spring-jdbc-2/pom.xml:1]: Add space between:
   </repositories>
    <pluginRepositories>
    
Add space between:
        </build>
    <repositories>
    
    Remove space between:
        <modelVersion>4.0.0</modelVersion> and
    <groupId>org.example</groupId>

File: quarkus-modules/quarkus-funqy/pom.xml [hunk 4]
```diff
@@ -13,6 +13,7 @@
         <artifactId>quarkus-modules</artifactId>
```
Comment (anastasiosioannidis) [inline quarkus-modules/quarkus-funqy/pom.xml:13]: Add spaces between the elements

File: persistence-modules/core-java-persistence-3/pom.xml [hunk 5]
```diff
@@ -1,4 +1,6 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```
[the remark this answers is not in the document]
Comment (Bipinkumar27) [reply persistence-modules/core-java-persistence-3/pom.xml:1]: done

File: persistence-modules/spring-data-jpa-repo-3/pom.xml (hunk 2 above)
Comment (Bipinkumar27) [reply persistence-modules/spring-data-jpa-repo-3/pom.xml:3]: done

File: persistence-modules/spring-jdbc-2/pom.xml (hunk 3 above)
Comment (Bipinkumar27) [reply persistence-modules/spring-jdbc-2/pom.xml:1]: done

File: quarkus-modules/quarkus-funqy/pom.xml (hunk 4 above)
Comment (Bipinkumar27) [reply quarkus-modules/quarkus-funqy/pom.xml:13]: done

**Pull request merged**